Adding Luna docs (#204)
rungalileo/docs-official: concepts/luna/luna.mdx

---
title: Luna 2
description: Discover Galileo's Luna 2 Evaluation model, reducing the latency and cost for metric evaluations
---

> **Luna 2** is the latest generation Galileo **Evaluation Foundation Model**, a small language model (SLM) that has been fine tuned to provide low latency and reduced costs for metric evaluations. Luna is designed to be further fine tuned for your specific use cases and custom metrics.

Reviewer: ```suggestion
**Luna 2** is the latest generation of our Luna small language models (SLMs), purpose built for scaling AI evaluations. Luna models are fine tuned to provide low latency and reduced costs for metric evaluations. Luna is designed to be further fine tuned for your specific use cases and custom metrics with the goal of providing scalable, real-time, customizable evaluations for enterprises. 
```

Reviewer: We're going away from EFMs as a phrase

> Luna-based metrics offer highly accurate and efficient evaluations for generative AI applications, particularly those with agentic workflows.

Reviewer: ```suggestion
Luna-based metrics offer highly accurate and efficient evaluations for AI applications, particularly those with agentic workflows.
```

Reviewer: Suggest broadening

<Note>
Luna is only available in the Enterprise tier of Galileo. [Contact us](https://galileo.ai/contact-sales) to learn more and get started.
</Note>

<CardGroup cols={2}>
<Card title="Contact us" icon="comment" horizontal href="https://galileo.ai/contact-sales">
    Contact us to learn more about using Luna in your evaluations
</Card>
<Card title="Read our research" icon="brain" horizontal href="https://galileo.ai/research">

>     Learn how Galileo pushes the envelope on GenAI evaluation with our family of Evaluation Foundation Models

Reviewer: ```suggestion
    Learn how Galileo pushes the envelope on GenAI evaluation with our family of fine tuned small language models.

</Card>
</CardGroup>

## Overview

> LLMs are powerful judges for evaluations, but as your application scales up to from tens or hundreds of traces a day, to thousands or millions, they can fall short.

Reviewer: ```suggestion
LLMs are powerful judges for evaluations, but as your application scales up to from tens or hundreds of traces a day, to thousands or millions, they can fall short. Too often, organizations relying solely on LLMs to act as judges incur major inference costs and don't see the low-latency they need to enable real-time evaluations and guardrailing. 
```

- **Enhanced Accuracy:** Luna 2 demonstrates at least a 10% accuracy increase compared to traditional BERT-based models, perfect for precise monitoring in production environments.

## Get started with Luna 2

If you are using the enterprise tier of Galileo, follow these steps to use Galileo's Luna-based metrics:

1. [Contact Galileo's customer support or account management](https://galileo.ai/contact-sales) to begin onboarding.
1. If you are using a Galileo-hosted instance, request L4 GPUs or higher, necessary for running Luna models. Otherwise you can deploy to your own infrastructure, using L4 or higher GPUs.
1. Review the provided documentation and model cards for details on latency, accuracy, and comparisons to BERT-based metrics.
1. Provide Galileo with relevant labelled sample data to fine tune the model. We can augment this with synthetic data if needed.
1. Galileo will fine tune your model for you, and deploy it.
1. Set up your experiments and log streams to use [Luna-based metrics](/concepts/metrics/metrics-reference#luna-metrics).

> This is not a one-shot process. Your model can be tuned on a regular basis as required.

Reviewer: Think we should describe this process or link to relevant doc sections here? CLHF maybe?

Reviewer: Makes sense to me! Thanks Jim

Author: Actually - added to this PR!
